jobs/release.Jenkinsfile

```
@@ -111,7 +111,11 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
         assert builtarches.contains("x86_64"): "The x86_64 architecture was not in builtarches."
         if (!builtarches.containsAll(basearches)) {
             if (params.ALLOW_MISSING_ARCHES) {
-                echo "Some requested architectures did not successfully build! Continuing."
+                // Display a warning, but continue.
+                def message = "Some requested architectures did not successfully build!"
+                warnError(message: message) {
+                    error(message)
+                }
```

Feels like we're using this API wrong. I think this is equivalent to just:

```suggestion
                echo "Some requested architectures did not successfully build!"
                currentBuild.result = 'UNSTABLE'
```

---

Not exactly. With `warnError` the message gets highlighted in the web view versus you having to search for it. See the following examples:

- `warnError()`: [link](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/release/detail/release/271/pipeline)
- `result='UNSTABLE'`: [link](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/release/detail/release/272/pipeline/)